styleguide/global/global.js

```diff
@@ -78,7 +195,10 @@ jQuery(document).ready(($) => {
 
   // Zoolander only solution for auto collapsing menu on example pages
   const url = window.location.pathname;
-  if ((url.match(/\/derek\/incubation\//gi) && url !== '/derek/incubation/') || (url.match(/\/derek\/solutions\//gi) && url !== '/derek/solutions/') || (url.match(/\/derek\/templates\//gi) && url !== '/derek/templates/')) {
+  if ((url.match(/\/derek\/incubation\//gi) && url !== '/derek/incubation/') ||
+      (url.match(/\/derek\/view-templates\//gi) && url !== '/derek/incubation/') ||
+      (url.match(/\/derek\/solutions\//gi) && url !== '/derek/solutions/') ||
+      (url.match(/\/derek\/templates\//gi) && url !== '/derek/templates/')) {
```

Comment:
That said, this doesn't appear to be working for me. It's staying open when I load it.
![image](https://cloud.githubusercontent.com/assets/5263371/24564374/42122a34-1617-11e7-9095-8879ad9fd9fa.png)

Also that second part of the URL doesn't make sense here: 
```
(url.match(/\/derek\/view-templates\//gi) && url !== '/derek/incubation/') 
```